frontend/device/android/device.lua

```diff
@@ -51,4 +51,16 @@ function Device:init()
     Generic.init(self)
 end
 
+function Device:initNetworkManager(NetworkMgr)
+    NetworkMgr.turnOnWifi = function()
+        android.setWifiEnabled(true)
+    end
+
+    NetworkMgr.turnOffWifi = function()
+        android.setWifiEnabled(false)
+    end
+end
+
+
```

Comment:
Stray newlines.

Reply:
removed but does it have any influence to the app? Or is just a Koreadernon-sense